presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraSession.java

```diff
@@ -94,35 +94,35 @@ public Session load(String key)
                 });
     }
 
-    public Set<Host> getReplicas(final String schemaName, final ByteBuffer partitionKey)
+    public Set<Host> getReplicas(String schemaName, ByteBuffer partitionKey)
     {
-        return executeWithSession(schemaName, session -> session.getCluster().getMetadata().getReplicas(schemaName, partitionKey));
+        return executeWithSession(session -> session.getCluster().getMetadata().getReplicas(schemaName, partitionKey));
     }
 
-    private Session getSession(String schema)
+    private Session getSession()
     {
         try {
-            return sessionBySchema.get(schema);
+            return sessionBySchema.get("");
```

Comment:
Are you positive we use the fully qualified tablename in all queries? (see https://docs.datastax.com/en/drivers/java/2.0/com/datastax/driver/core/Session.html)